index.html

```diff
@@ -165,6 +169,19 @@ <h3>For argon2</h3>
                 var optionsLink                   = document.querySelector('a.options');
                 var optionsDiv                    = document.querySelector('div.options');
 
+                var displayInfos = function(e) {
+                    if(e) e.preventDefault();
+                    infosDiv.classList.remove('hidden');
+                    displayInfosLink.classList.add('hidden');
+                }
+
+                if (window.location.search.indexOf('pwa') !== -1) {

```

Comment:
So, if using hash, it would be `window.location.hash.indexOf('pwa')`

Reply:
done